### DISCUSSION TITLE
Ability to specify files/folders for backups

### DISCUSSION THREAD

**Discussion by HardGaming-tech** [pelican-dev/panel#34] [Ideas]
currently the backup solution is fairly heavy. it get 5gb of files that are generally not necessary and are part of the game servers. this increase how much storage is used and decrease time to restore a game server. 

if possible i would like a way for us to specify WHAT to backup for the EGG as opposed to it just getting the entire container.

**Comment by purejosh** [comment]
You're able to use the exclude feature to accomplish this. This suggestion is simply the inverse of what already exists in Pterodactyl. Baking it into the egg is a weird idea, but that's a feature that doesn't exist at this time.

**Comment by trenutoo** [comment]
This is already possible. Pterodactyl/Pelican supports uploading a `.pteroignore` file in the root directory of the server, which follows the .gitignore format or you can specify the rules directly when creating a backup manually. You can ignore everything and whitelist specific folders/files that you'd want to backup.

`#` are comments.
```
# use * to ignore everything

* 

# After ignoring everything, use ! to unignore/whitelist files or paths for a backup

# whitelist all files and subfolders inside config folder
!config/

# whitelist specific file
!config/config.cfg

**Comment by Boy132** [comment]
Like Software-Noob said it's already possible. With Pelican the file is called `.pelicanignore`.
